File: Python/D6 - T/2 - sum of digits of num/2.py
# Write a python program to print the sum of the digits of the given number.
#  for example: input = 135, ans = 9

num = int(input('Enter the number : '))
digit = 0
sum = 0

while(num != 0):
    digit = num % 10
    sum = sum + digit
    num = num // 10

print(sum)

File: C/D4/2 - sum of series2/2.cpp
// Write a C program to find the total sum of each number power with its own, up to given number.
// 1  raised to 1 + 2 raised to 2 + 3 raised to 3 + 4 raised to 4 + ....

#include <stdio.h>
#include <math.h>

int main(){
    int i, num, ans = 0;

    printf("Enter a number : ");
    scanf("%d", &num);

    for(i=1; i<=num; i++){
        ans = ans + pow(i, i);
    }
    printf("Total sum is %d", ans);
}